### ISSUE TITLE
RFC: peer-star vs. peer-base

### ISSUE THREAD

**Issue by pgte** [ipfs/dynamic-data-and-capabilities#56]
Now that we've migrated peer-star-app into peer-base, we need to understand what to make of peer-star.

Here is the way that I see this:

## peer-base is a library

Peer-base is a library that helps you create DApps on top of IPFS. It will contain a main library (peer-base) and can contain libraries that peer-base depend on (like peer-base-discovery, for instance).

## peer-star are DApp blueprints

"peer-star" stands for "peer-*" (peer-pad, peer-chess, etc.) and depends on peer-base.

Peer-star is where the blueprints for how to build DApps live. This should be the place for peer-pad, Discussify, the peer-base to-do list, etc.

Please tell me your thoughts on this.

**Comment by dirkmc** [comment]
@pgte are you considering these names in a marketing sense, as in "I'm building my app on peer-base" or in terms of a github structure, as in "the code is at https://github.com/peer-base/peer-chess"?

Before I knew what peer-star was I had some trouble finding it, because when referring to the projects in a marketing sense, some people would write peer-* and some would write peer-star. So then when searching for the projects on github I wasn't sure what to search for.

**Comment by pgte** [comment]
@dirkmc so far the marketing plans on a nutshell are:

* peer-base to have its own website (and logo, etc.), containing API and references to existing blueprints, projects (@marcooliveira is figuring this out as we speak).
* peer-star is not really a place any more. We chose this to not confuse newcomers any more. We can redirect peer-star to peer-base. Not a final decision on this.

@dirkmc thoughts on this?

**Comment by dirkmc** [comment]
Ah I see, that makes sense, I think peer-base is a clearer name for newcomers 👍 

My point of confusion is `Peer-star is where the blueprints for how to build DApps live` - what do you mean by "where they live"?

**Comment by jimpick** [comment]
I agree with building the branding around "peer-base" ... if people only learn one word, it should be that.

Internally, we can still use "peer-*" when we need a glob for conversation. :-)

**Comment by pgte** [comment]
@dirkmc yes, I agree, that's a bit confusing, as there's no official "home" for peer-* apps.
We plan to, when having a peer-base website, to list some blueprints there.

Perhaps we should, as @jimpick suggested, to stop talking about peer-* so that we don't confuse anyone else :)
